### DISCUSSION TITLE
[Improvement]: Port mapping refactor

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#3161] [Improvement Requests]
### Description

Por mapping should be simmilar to the environment variables UI

### Minimal Reproduction (if possible, example repository)

Improvement ideas:

- [ ] Let us map Ports with the protocol -> UDP or TCP: 80:8080/TCP or 80:8080/UDP
- [ ] Please let us still use rolling updates even with port mapping

### Version

-

**Comment by alexandrucancescu** [comment]
Also interested in mapping UDP ports

**Comment by bobmoff** [comment]
Yeah, havent figured out how to setup UDP portforwarding without opening up ports directly to the host.

**Comment by plutack** [comment]
This is already implemented right? Can we have this added to the documentation?
